pkg/migration/templates/controlplane_azure_machine_template.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureMachineTemplate
+metadata:
+  name: {{.ClusterID}}-control-plane
+  namespace: default
+spec:
+  template:
+    spec:
+      dataDisks:
+      - diskSizeGB: 256
+        lun: 0
+        nameSuffix: etcddisk
+      enableIPForwarding: true
+      identity: SystemAssigned
+      location: germanywestcentral
```

Comment:
Is fixed location here ok?

Reply:
Wuups, nope. Gotta pull it from `AzureCluster`. Good catch! Thanks! :+1: